### ISSUE TITLE
Benefits Hub audit

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17366]
## Description
Complete a full accessibility audit of the Benefits Hub product, including:
- Landing pages
  - https://www.va.gov/health-care/
  - https://www.va.gov/education/
- Detail pages
  - https://www.va.gov/health-care/about-va-health-benefits/
  - https://www.va.gov/disability/eligibility/

## Acceptance Criteria

- [ ] Accessibility audit has been completed
- [ ] Tickets have been created for recommended updates and improvements
- [ ] Any high priority/critical updates are brought to the attention of PM/DM/PO
- [ ] Design review ticket is created

**Comment by laflannery** [comment]
### Accessibility Updates

- Consider updating the Vet portraits at the bottom of landing pages to a background image
- Secondary nav
  - On pages without this, the section is still in the DOM but empty. This needs to be conditional so it doesn’t appear at all. (e.g. Accessibility, Harassment)
- On this page component
  - [Focus management issue on mobile](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1979)
  - [An incorrect ARIA role is being used](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1998)

### Updates that should be made to align with upcoming DST changes:

**`<va-icon>` migration**
Planned for deprecation of current icons beginning of May
- Icons for the names of each hub. Displayed on:
  - Landing pages
  - Detail pages where there is a left nav
- Landing page Connect with us section
  - One of these is the “channel” prop in the <va-link> component but the rest aren’t. Why do we have a youtube version but not the other social media channels?

**Upcoming Heading/Link guidance**
- Update sidebar promo to align with new guidance
  - There is already [a ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12366) in the backlog
- Complementary info section
  - This is using the Related links “component” however we have heading links and I’m not sure the “component” does. Even if it does, this wouldn’t fit with the upcoming guidance so should this be rethought?
  - Examples - Harassment, Check your COE Status
- Link Teaser blocks
  - These creak heading links. Should these be reconsidered?
  - They are in use on the [About Gi Bill Benefits page](https://www.va.gov/education/about-gi-bill-benefits/) and [VR&E page](https://www.va.gov/careers-employment/vocational-rehabilitation/). Probably other pages as well

**Components**
- [Language toggle](https://design.va.gov/components/language-toggle)
  - The plan is to build this and have it available for use within the next month or two. There are [tickets](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/667) for this in the DST board

### Other updates/improvements:
- Sidenav
  - This is a whole thing and we probably aren’t going to do anything about it right now
